ansible/roles/etcd/tasks/etcd2-configure.yml

```diff
@@ -11,10 +11,3 @@
 - name: CoreOS | Write etcd config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify: restart etcd2
```

Comment:
Ended up making a new play file for `etcd_restart.yml` and `etcd2_restart.yml` since I am conditionally calling them once at the end of main and not using it like handlers should be used.